src/styles/Homepage.styles.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import colors from "./colors";
+import fonts from "./font_names";
+
+export const HeroContainer = styled.div`
+  background-size: cover;
+  background-color: black;
+`;
+
+export const HeroContent = styled.div`
+  position: relative;
+  z-index: 1;
+
+  padding-top: 233px;
+  margin-left: 6vw;
+  img {
+    width: 50vw;
+  }
+`;
+
+export const HeroDescription = styled.div`
+  font-family: itc-avant-garde-gothic-pro, sans-serif;
+  font-size: 27px;
+  font-weight: 500;
+  width: 34vw;
+  color: ${colors.white};
+  margin-top: 39px;
+
+  font-weight: 300;
+`;
+
+export const MissionSection = styled.div`
+  margin-top: 367px;
+  display: flex;
+`;
+
+export const MissionTitle = styled.div`
+  font-family: itc-avant-garde-gothic-pro, sans-serif;
+  font-size: 60px;
```

Comment:
same as above - also if you want to start migrating your font sizes from `px` to `ems` now this will be easier to do earlier on!